main.py

```diff
@@ -1,7 +1,58 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+
+    Genre.objects.create(
+        name="Western"
+    )
+    action = Genre.objects.create(
+        name="Action"
+    )
+    dramma = Genre.objects.create(
+        name="Dramma"
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama' to maintain consistency with the later update on line 38.